docs/howto/writing-migrations.txt

```diff
@@ -183,3 +183,47 @@ the respective field according to your needs.
   Note there is a race condition if you allow objects to be created while this
   migration is running. Objects created after the ``AddField`` and before
   ``RunPython`` will have their original ``uuid``’s overwritten.
+
+Controlling the order of migrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Django controls the order of migrations not by reference to the filename
+of each migration, but by building a graph using two properties on the
+``Migration`` class. These properties are ``dependencies`` and
+``run_before``.
+
+If you've used the :djadmin:`makemigrations` command then you've
+probably already seen ``dependencies`` in action because auto-created
+migrations have this defined as part of their creation process.
+
+The ``dependecies`` property is declared like this::
+
+    # -*- coding: utf-8 -*-
+    from django.db import migrations
+
+    class Migration(migrations.Migration):
+
+        dependencies = [
+            ('myapp', '0123_the_previous_migration'),
+        ]
+
+Usually this will be enough, but from time to time you may need to
+ensure that your migration runs *before* other migrations. This is
+useful, for example, to make third-party apps' migrations run *after*
+your :setting:`AUTH_USER_MODEL` replacement.
+
+To achieve this, state those migrations before which your own must run
```

Comment:
It's correct, normal English. I have given some thought to rewording it but other versions are either too verbose or incorrect in grammar.